build/repo.targets

```diff
@@ -325,7 +330,143 @@
+      <RuntimeStoreNoTimestampRPMFileName>$(RuntimeStoreNoTimestampInstallerPackageName)-$(InstallerPlatform)-$(PACKAGE_CACHE_PLATFORM).rpm</RuntimeStoreNoTimestampRPMFileName>
+      <RPMDirectoriesArguments>@(RPMDirectories->' --directories %(FullPath)', ' ')</RPMDirectoriesArguments>
+      <InstallerSummary>Microsoft ASP.NET Core Runtime Package Store 2.0.0 - Preview 2</InstallerSummary>
+      <InstalerDescription>Runtime package store for Microsoft ASP.NET Core. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/aspnet/home). We happily accept issues and PRs.</InstalerDescription>
+      <InstallerMaintainerName>Microsoft</InstallerMaintainerName>
+      <InstallerMaintainerEmail>nugetaspnet@microsoft.com</InstallerMaintainerEmail>
+      <InstallerVendor>.NET Foundation</InstallerVendor>
+      <InstallerLicense>Apache</InstallerLicense>
+      <InstallerPackageRevision>1</InstallerPackageRevision>
+      <InstallerHomepage>https://www.asp.net/</InstallerHomepage>
+      <InstallerInstallRoot>/usr/share/dotnet</InstallerInstallRoot>
+    </PropertyGroup>
+
+    <!-- Clear the output dir -->
+    <RemoveDir Directories="$(RuntimeStoreInstallerOutputTimeStampDirectory)" />
+    <RemoveDir Directories="$(RuntimeStoreInstallerOutputNoTimeStampDirectory)" />
+    <MakeDir Directories="$(RuntimeStoreInstallerOutputTimeStampDirectory)" />
+    <MakeDir Directories="$(RuntimeStoreInstallerOutputNoTimeStampDirectory)" />
+
+    <!-- Build Docker Image -->
+    <Exec
+      Command="docker build --build-arg USER_ID=%24(id -u) -t docker-image-$(InstallerPlatform) $(InstallerPlatform)"
+      WorkingDirectory="$(DockerDir)" />
+
+    <!-- Build Runtime Store Deb package -->
+    <Exec Command="docker run
+      --rm
+      -v %24(pwd):$(DockerRootDirectory)
+      docker-image-$(InstallerPlatform)
+      fpm
+        --verbose
+        -s dir
+        -t rpm
+        -n $(RuntimeStoreTimestampInstallerPackageName)
+        -v $(TimestampVersion)
+        -a amd64
+        $(RPMDirectoriesArguments)
+        --rpm-changelog $(DockerRootDirectory)/artifacts/itLayout/templates/changelog
+        --rpm-summary &quot;$(InstallerSummary)&quot;
+        --description &quot;$(InstalerDescription)&quot;
+        --maintainer &quot;$(InstallerMaintainerName) %3C$(InstallerMaintainerEmail)%3E&quot;
+        --vendor &quot;$(InstallerVendor)&quot;
+        -p $(DockerRootDirectory)/artifacts/itOutput/$(RuntimeStoreTimestampRPMFileName)
+        --license &quot;$(InstallerLicense)&quot;
+        --iteration $(InstallerPackageRevision)
+        --url &quot;$(InstallerHomepage)&quot;
+        $(DockerRootDirectory)/artifacts/itLayout/package_root/=$(InstallerInstallRoot)"
+      WorkingDirectory="$(RepositoryRoot)"/>
+    <Exec Command="docker run
+      --rm
```

Comment:
Does this work (having the command split across multiple lines)? Asking because this doesn't work on WIndows (the console host treats each line a separate command).